html/apps/barchart/services/analyticService.js

```diff
@@ -0,0 +1,202 @@
+angular.module('myApp')
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: ["angular" is not defined.](https://www.codacy.com/app/sudheesh1995/loklak_server/file/2626095151/issues/source?bid=3288751&fileBranchId=3313532#l1)